brokkr/brokkr.mk

```diff
@@ -30,6 +30,9 @@ $(_BROKKR_PLUGINS_MK): $(_BROKKR_PLUGINS_SENTINEL)
 		url="https://raw.githubusercontent.com/$(BROKKR_REPO)/$${plugin_version}/plugins/$${plugin_path}.mk"; \
 		echo "Downloading $${url}"; \
 		curl --fail -s "$$url" -o $(_BROKKR_PLUGINS_DIR)/$${plugin_path}.mk; \
+		url="https://raw.githubusercontent.com/$(BROKKR_REPO)/$${plugin_version}/plugins/$${plugin_path}.sh"; \
+		echo "Downloading $${url}"; \
+		curl --fail -s "$$url" -o $(_BROKKR_PLUGINS_DIR)/$${plugin_path}.sh; \
```

Comment:
Ha! Smart

Reply:
I like it

Reply:
How did I make a release agian?
Manually?

Reply:
Yeah, unfortunately no README or fancy routine. Create a release and use the branch, or merge with master and release from that.